app/views/reports/report_month.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= link_to "back", reports_path, class: "btn btn-info" %>
+
+<table class="table">
+    <thead class="thead-ligth">
+      <tr>
+        <th class="col">Id</th>
+        <th class="col">Check In</th>
+        <th class="col">Check Out</th>
+        <th class="col">Employee Id</th>
+        
+      </tr>
+    </thead>
+    <tbody>
+      <% 
+        @reports = Report.all 
+        reports_today = 0
+      %>
+      <% @reports.each do |report| %>
+        <% 
+          today_date = Time.now
+          date_report = report.check_in
+        %>
+        <% if today_date.month == date_report.month && today_date.year == date_report.year%>
+          <tr>
+            <th scope="row">
+              <%= report.id %>
+            </th>
+            <td>
+              <%= "#{report.check_in.strftime('%d-%B-%Y')}  #{report.check_in.strftime('%R')} "%>
+            </td>
+            <td>
+              <%= 
+              if report.check_out == nil
+                report.check_out
+              else
+                "#{report.check_out.strftime('%d-%B-%Y')}  #{report.check_out.strftime('%R')} " 
+              end
+              %>
+            </td>
+            <td>
+              <%= report.employee_id %>
+            </td>
+          </tr>
+        <%else%>
+
+        <%end%>          
+      <% end %>
+      <%if reports_today == 0%>
+        <h1>No reports today</h1>
+      <%end%>
+    </tbody>
+  </table>
```

Comment:
Add a blank line.

Reply:
thanks for your feedback, we are  going to correct that